src/main/java/fr/zenity/desbugs/Enum/UrlEnvironment.java

```
@@ -38,23 +38,33 @@ public static UrlEnvironment initCustom(String env){
         return build(env,true);
     }
 
-    public void setUrls(String urlsProperties){
-        Map<String, String> urls = Splitter.on(",").withKeyValueSeparator("=").split(urlsProperties.replaceAll("\\s+", ""));
-        urlLanding = urls.get("urlLanding");
-        urlApp = urls.get("urlApp");
+    public void setUrls(String urlsProperties) {
+
+        if(urlsProperties.equals("null"))
+            urlLanding = urlApp = null;
```

I think, the urls should never equal "null". Should we return an error in this case ?